spec/Section 5 -- Validation.md

```diff
@@ -1873,6 +1873,8 @@ variable.
 
 IsVariableUsageAllowed(variableDefinition, variableUsage):
 
+- If {variableUsage} is nested within a custom scalar literal value, return
+  {true}.
 - Let {variableType} be the expected type of {variableDefinition}.
 - Let {locationType} be the expected type of the {Argument}, {ObjectField}, or
   {ListValue} entry where {variableUsage} is located.
```

Comment:
Rather than enforcing that implementations MUST NOT validate variables within custom scalars, as this edit does, let's instead give them the choice:

```suggestion
- Let {variableType} be the expected type of {variableDefinition}.
- Let {locationType} be the expected type of the {Argument}, {ObjectField}, or
  {ListValue} entry where {variableUsage} is located.
- If {locationType} cannot be determined, for example when {variableUsage} is
  nested within a custom scalar literal value, return {true}.
```

Reply:
It's technically the same? If `variableUsage` is nested within a literal value, then `{locationType}` cannot be determined and we return `true` ? At least in the "graphql type" acceptation. 

The custom scalar may expect some kind of literal but it's not a GraphQL type per-se. I think if we want to allow validating variables within custom scalars, we'll need the coercing (see also [the comment below](https://github.com/graphql/graphql-spec/pull/1156#discussion_r2027346387))?